src/components/clients/Form.js

```diff
@@ -1,15 +1,17 @@
-import { Input, Select } from "@chakra-ui/react";
```

Comment:
This did not work correctly on Create form (at least in my case). 
In this scenario, we do not have any contact details, yet. But still the Select loads "General" as the selected value. Evenmore, when I changed it to a different value, it did not send it to the API. So, there was an error in logs and the contact creation has failed.

Reply:
as we discussed offline, this is not relevant anymore :)